futures/tests/ready_queue.rs

```diff
@@ -1,18 +1,15 @@
-mod assert_send_sync {
-    use futures::stream::FuturesUnordered;
-
-    pub trait AssertSendSync: Send + Sync {}
-    impl AssertSendSync for FuturesUnordered<()> {}
-}
```

Comment:
This module was just removed because covered by https://github.com/rust-lang/futures-rs/pull/2323.